lib/store_model/nested_attributes.rb

```diff
@@ -13,22 +13,32 @@ module ClassMethods # :nodoc:
       # @param associations [Array] list of associations and options to define attributes, for example:
       #   accepts_nested_attributes_for [:suppliers, allow_destroy: true]
       #
+      # Alternatively, use the standard Rails syntax:
+      #
+      # @param associations [Array] list of associations and attributes to define getters and setters.
+      #
+      # @param options [Hash] options not supported by StoreModel will still be passed to ActiveRecord.
+      #
       # Supported options:
       # [:allow_destroy]
       #   If true, destroys any members from the attributes hash with a
       #   <tt>_destroy</tt> key and a value that evaluates to +true+
       #   (e.g. 1, '1', true, or 'true'). This option is off by default.
       def accepts_nested_attributes_for(*associations)
```

Comment:
Rubocop is still mad about this one 🤔 Do you have an idea how to reduce the complexity here? I'd be happy with `disable` as well 🙂

Reply:
Updated so that rubocop doesn't complain about anything, no `disable` required.